app/views/shared/partials/_dates_form.html.erb

```diff
@@ -1,6 +1,8 @@
 <% fields ||= ['date_start', 'date_end'] %>
 <% labels ||= { date_start: "Startdatum *", date_end: "Enddatum" } %>
 <% list_of_dates = record.dates.presence || [OpenStruct.new] %>
+<% date_end ||= nil %>
+<% requireds ||= [] %>
```

Comment:
hab das unter `fields` geschoben weil es da thematisch mehr hinpasst und auch in den partial calls von dir an die stelle gepackt wurde. liest sich einheitlicher